app/controllers/hyrax/admin/collection_type_participants_controller.rb

```diff
@@ -1,13 +1,18 @@
+# frozen_string_literal: true
+
 # OVERRIDE FILE from Hryax v2.9.0
-require_dependency Hyrax::Engine.root.join('app', 'controllers', 'hyrax', 'admin', 'collection_type_participants_controller').to_s
+require_dependency Hyrax::Engine.root
+                                .join('app', 'controllers', 'hyrax', 'admin', 'collection_type_participants_controller')
+                                .to_s
```

Comment:
This isn't terrible by any means, but could probably be a bit more readable. Let's link up on this one.